src/frontend/Komponenter/Personoversikt/Dokumenter.tsx

```diff
@@ -153,93 +262,160 @@ const Dokumenter: React.FC<{ fagsakPerson: IFagsakPerson }> = ({ fagsakPerson })
                 </InnUt>
             </Td>
             <Td>
-                <HovedLenke
-                    key={dokument.journalpostId}
-                    href={`/dokument/journalpost/${dokument.journalpostId}/dokument-pdf/${dokument.dokumentinfoId}`}
-                    target={'_blank'}
-                    rel={'noreferrer'}
-                >
-                    {dokument.tittel}
-                </HovedLenke>
-                <LogiskeVedlegg logiskeVedlegg={dokument.logiskeVedlegg} />
+                {(toggles[ToggleName.dokumentoversiktLinkTilDokument] || dokument.tema === 'ENF') &&
+                    dokument.harSaksbehandlerTilgang && (
+                        <>
+                            <HovedLenke
+                                key={dokument.journalpostId}
+                                href={`/dokument/journalpost/${dokument.journalpostId}/dokument-pdf/${dokument.dokumentinfoId}`}
+                                target={'_blank'}
+                                rel={'noreferrer'}
+                            >
+                                {dokument.tittel}
+                            </HovedLenke>
+                            <LogiskeVedlegg logiskeVedlegg={dokument.logiskeVedlegg} />
+                        </>
+                    )}
+                {((!toggles[ToggleName.dokumentoversiktLinkTilDokument] &&
+                    dokument.tema !== 'ENF') ||
+                    !dokument.harSaksbehandlerTilgang) && (
+                    <BodyShortSmall>{dokument.tittel}</BodyShortSmall>
+                )}
```

Comment:
hvis vi legger til
```
const skalViseLenke =
            (toggles[ToggleName.dokumentoversiktLinkTilDokument] || dokument.tema === 'ENF') &&
            dokument.harSaksbehandlerTilgang;
```
tror jeg dette blir ryddigere og ekvivalent 🤓 
```suggestion
                {skalViseLenke ? (
                        <>
                            <HovedLenke
                                key={dokument.journalpostId}
                                href={`/dokument/journalpost/${dokument.journalpostId}/dokument-pdf/${dokument.dokumentinfoId}`}
                                target={'_blank'}
                                rel={'noreferrer'}
                            >
                                {dokument.tittel}
                            </HovedLenke>
                            <LogiskeVedlegg logiskeVedlegg={dokument.logiskeVedlegg} />
                        </>
                    ) : (
                        <BodyShortSmall>{dokument.tittel}</BodyShortSmall>
                    )}
```

Reply:
Ja, det ble meget bedre!